### ISSUE TITLE
Please add ability to link to particular view

### ISSUE THREAD

**Issue by opsmekanix** [LinkedInAttic/white-elephant#13]
It would be nice to to share a URL which would zoom one into a particular data view.  Kind of like the 'share link' button with Google Maps.

**Comment by matthayes** [comment]
Good idea, this would be useful.  We could use the hash in the URL to represent the current view and update it as the view changes.
